src/main/java/org/jfrog/bamboo/configuration/ArtifactoryXrayScanConfiguration.java

```diff
@@ -59,4 +57,13 @@ public boolean taskProducesTestResults(@NotNull TaskDefinition taskDefinition) {
         return configMap;
     }
 
+    @Override
+    public void validate(@NotNull ActionParametersMap params, @NotNull ErrorCollection errorCollection) {
+        // Validate scan server.
+        String scanServerKey = XrayScanContext.SERVER_ID_PARAM;
+        if (StringUtils.isBlank(params.getString(scanServerKey))) {
+            errorCollection.addError(scanServerKey, "Please specify a Server.");
+        }
```

Comment:
Can we move the above isBlank check into the *validateArtifactoryServer* mathod?
This comment applies to all the usages of the *validateArtifactoryServer* method.

Reply:
Created a new method - validateArtifactoryServerProvidedAndValid.
This one will also make sure that a server is provided, and if provided - that it is valid.